testutil/testnode/full_node_test.go

```diff
@@ -96,7 +96,7 @@ func TestIntegrationTestSuite(t *testing.T) {
 func (s *IntegrationTestSuite) Test_FillBlock() {
 	require := s.Require()
 
-	for squareSize := 2; squareSize < appconsts.DefaultMaxSquareSize; squareSize *= 2 {
+	for squareSize := 4; squareSize < appconsts.DefaultMaxSquareSize; squareSize *= 2 {
```

Comment:
Why did you change this? Just to make the tests quicker?

Reply:
originally, yeah just to simplify the test, and I don't think anyone is using this to fill the square to size 2. Still, in retrospect it was the wrong move, and I should have just adjusted the fill block test to account for this...

However now we can't actually have square sizes of 2 anymore after the adjustment as we're adding 3 shares of padding to all estimates.